src/sql/src/rbac.rs

```diff
@@ -284,6 +289,8 @@ pub fn check_usage(
     resolved_ids: &ResolvedIds,
     item_types: &BTreeSet<CatalogItemType>,
 ) -> Result<(), UnauthorizedError> {
+    rbac_preamble!(catalog, session);
```

Comment:
Does this need to be a macro, or could it be a function? Definitely not blocking but a function like `check_rbac_preamble(catalog, session)?;` might be a bit cleaner

Reply:
Done